dhall-yaml/release.nix

```diff
@@ -0,0 +1,8 @@
+let
+  default = (import ./default.nix);
+
+in
```

Comment:
You don't need to create a `release.nix` for `dhall-yaml`.  The `release.nix` underneath the `dhall-json` directory is a relic from back when it used to be a separate repository but the file is no longer used.  The correct thing to do is to delete `dhall-json/release.nix` instead

Reply:
Ah, good. Fixed! :)